pkg/helm/charts.go

```diff
@@ -0,0 +1,201 @@
+package helm
+
+import (
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"os"
+	"path/filepath"
+	"regexp"
+	"strings"
+
+	"github.com/Masterminds/semver"
+	"sigs.k8s.io/yaml"
+)
+
+// Load opens a Chartfile tree
+func Load(projectRoot string) (*Charts, error) {
```

Comment:
`Load` seems a bit generic. Can we have `LoadChartFile`? Then `helm.LoadChartFile()` really tells us what it is doing.